### PR TITLE
Filter some validation when installed from a package

### PR DESCRIPTION
**Pull request by dlangille**
Also, tell the user that fping FAILURES can be ignored if running LibreNMS in a
jail without ::1

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/Validations/Programs.php [hunk 1]
```diff
@@ -97,6 +97,7 @@ public function extraFpingChecks(Validator $validator, $cmd)
 
     private function failFping($validator, $cmd, $output)
     {
+        $validator->warn("fping FAILURES can be ignored if running LibreNMS in a jail without ::1. You may want to test it manually: fping ::1");
```
Comment (Jellyfrog) [inline LibreNMS/Validations/Programs.php:100]: this should be info()

Comment (dlangille) [reply LibreNMS/Validations/Programs.php:100]: I'm stuck. I committed https://github.com/dlangille/librenms/commit/f6684161c31f89a288e8ea104d74ea5efd1802d1 and now I can't see how to push it here.

Comment (Jellyfrog) [reply LibreNMS/Validations/Programs.php:100]: you commit in the same branch and it happens automatically

Comment (dlangille) [reply LibreNMS/Validations/Programs.php:100]: Did that, and the automatic part did not happen.

Comment (murrant) [reply LibreNMS/Validations/Programs.php:100]: @dlangille your last commit changes it back to warn from info.  Either revert that or just make a new commit changing it to info.

Comment (dlangille) [reply LibreNMS/Validations/Programs.php]: Ouch. Repaired.

**Comment by murrant** [comment]
Just some style issues left now, you can download the diff from styleci to make it easy.

**Review by murrant — APPROVED** [review]

**Pull request merged**